### PR TITLE
Frontend for faucet app

### PR DESCRIPTION
**Pull request by romkaspb**

### CODE REVIEW

**Comment by goodsoft** [comment]
`Generate wallet` popup shows only an address and a public key.
We need to show private key too, for user to be able to restore this account in the wallet app.
Appropriate warnings about security should be present, I think.

File: faucet/frontend/src/locales/cn.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "welcomeMsg" : "在這裡你可以獲得令牌來測試DISCIPLINA區塊鏈",
```
Comment (volhovm) [inline faucet/frontend/src/locales/cn.json]: What a nice Chinese localisation :+1:

File: faucet/frontend/src/assets/api/student.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const _assignments = [
```
Comment (volhovm) [inline faucet/frontend/src/assets/api/student.js]: What do you need this stub data for?

File: faucet/frontend/src/locales/cn.json (hunk 1 above)
Comment (mkaito) [reply faucet/frontend/src/locales/cn.json]: I hope that doesn't accidentally tell someone to go pound sand.

**Pull request merged**